src/api/records.tsx

```diff
@@ -101,7 +101,7 @@ const fetchRecords = async (
   });
```

Comment:
I can't comment on the above - but I thought the whole point of the previous commit was to not send functions if they're not visible in the table? Yes we're now not projecting on them, but we're still sending the functions to the backend in the above and this still slows down the queries

Reply:
Ah, just realised that it was fixed in `export.tsx` but not in `records.tsx` - just need the same logic as exports I think